packages/msal-interactor/package.json

```diff
@@ -9,19 +9,20 @@
 	},
 	"repository": "https://github.com/microsoft/essex-js-toolkit",
 	"author": "Chris Trevino <chtrevin@microsoft.com>",
+	"contributors": [
+		"Chris Trevino <chtrevin@microsoft.com>",
+		"Derek Worthen <deworthe@microsoft.com>"
+	],
 	"license": "MIT",
 	"scripts": {
 		"build": "essex build",
 		"clean": "essex clean",
 		"watch": "essex watch"
 	},
-	"dependencies": {
-		"msal": "^1.4.2"
-	},
 	"devDependencies": {
 		"@essex/scripts": "^11.0.1"
 	},
-	"peerDependencies": {
```

Comment:
You may want to double check the dist/esm and dist/cjs output to verify that Babel is not weaving in core-js

Reply:
I don't think babel is weaving in core-js as I don't see any imports/requires for core-js and I don't see any polyfill definitions.